Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2137,11 +2143,11 @@ namespace Babylon
                 m_jsJointPose.Set("transform", m_jsTransform.Value());
             }
 
-            void Update(const Napi::Env& env, const xr::System::Session::Frame& frame, uint32_t timestamp)
+            void Update(const Napi::Env& env, std::shared_ptr<const xr::System::Session::Frame> frame, uint32_t timestamp)
```

Comment:
The member `m_frame` is a shared_ptr. it was the responsible raw pointer for the crash. It's not possible to assign a reference to a shared_ptr so it needs to be passed as a shared_ptr.

Reply:
I don't follow... anything can be passed as a ref or a const ref, right? In this case, it would just prevent unnecessary increments/decrements of the shared_ptr I would think.

Reply:
my bad, I misunderstood. yes, passing as const ref should be ok.